packages/mobile/src/components/challenge-rewards-drawer/ChallengeRewardsDrawerProvider.tsx

```diff
@@ -89,7 +89,7 @@ export const ChallengeRewardsDrawerProvider = () => {
   let progressRewardAmount = challenge?.totalAmount ?? 0
   if (challenge?.challenge_id === ChallengeName.OneShot) {
     progressRewardAmount =
-      (challenge?.claimableAmount ?? 0) + (challenge?.disbursed_amount ?? 0)
+      challenge?.claimableAmount || challenge?.disbursed_amount || 0
```

Comment:
nit: style guide says prefer `??` - does `challenge?.claimableAmount ?? chalenge?.disbursed_amount ?? 0` work?

Reply:
but watevs

Reply:
yeaaa it's different